include/openenclave/internal/sgxsign.h

```diff
@@ -61,6 +60,51 @@ oe_result_t oe_sgx_sign_enclave_from_engine(
     const char* key_id,
     sgx_sigstruct_t* sigstruct);
 
+/**
+ * Get the digest of an unsigned sigstruct given the expected MRENCLAVE,
+ * ATTRIBUTES, ISVPRODID, and ISVSVN values.
+ *
+ * @param mrenclave[in] hash of the enclave to be signed
+ * @param digest[out] the digest of the sigstruct to be signed
```

Comment:
missing @param lines for attributes, product_id, and security_version

Reply:
Fixed all comments in this file